modules/microshift-low-latency-prepare-workload.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-prepare-workload_{context}"]
+= Preparing a {microshift-short} workload for low latency
+
+To take advantage of low latency, workloads running on {microshift-short} must have the `microshift-low-latency` container runtime configuration set using the `RuntimeClass` feature. The CRI-O `RuntimeClass` object is installed with the `microshift-low-latency` RPM, so only the pod annotations need to be configured.
+
+.Prerequisites
+* You installed the `microshift-low-latency` RPM package.
+
+.Procedure
+
+. Use the following example to set the following annotations in the pod spec:
++
+[source,text]
+----
+cpu-load-balancing.crio.io: "disable"
+irq-load-balancing.crio.io: "disable"
+cpu-quota.crio.io: "disable"
+----
++
+.Example pod that runs `oslat` test:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: oslat
+  annotations:
+    cpu-load-balancing.crio.io: "disable" <1>
+    irq-load-balancing.crio.io: "disable" <2>
+    cpu-quota.crio.io: "disable" <3>
+spec:
+  runtimeClassName: microshift-low-latency <4>
+  containers:
+  - name: oslat
+    image: quay.io/container-perf-tools/oslat
+    imagePullPolicy: Always
+    resources:
+      requests:
+        memory: "400Mi"
+        cpu: "10"
+      limits:
+        memory: "400Mi"
+        cpu: "10"
+    env:
+    - name: tool
+      value: "oslat"
+    - name: manual
+      value: "n"
+    - name: PRIO
+      value: "1"
+    - name: delay
+      value: "0"
+    - name: RUNTIME_SECONDS
+      value: "60"
+    - name: TRACE_THRESHOLD
+      value: ""
+    - name: EXTRA_ARGS
+      value: ""
+    securityContext:
+      privileged: true
+      capabilities:
+        add:
+          - SYS_NICE
+          - IPC_LOCK
+
+----
+<1> Disables the CPU load balancing for the pod.
+<2> Opts the pod out of interrupt handling (IRQ).
+<3> Disables the CPU completely fair scheduler (CFS) quota at the pod run time.
+<4> The `runtimeClassName` must match the name of the performance profile configured in the cluster.
++
+[IMPORTANT]
+====
+For the pod to have the `Guaranteed` QoS class, it must have the same values of CPU and memory in requests and limits. See link:https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/#guaranteed[Guaranteed] (Kubernetes upstream documentation)
+====
+
+. To apply changes, restart {microshift-short}.
```

Comment:
Which changes would we be applying by restarting MicroShift for the workload?

Reply:
We should remove this. At this point all the configurations requiring reboot or service restart should be completed